reorder_python_imports.py

```diff
@@ -520,29 +520,39 @@ def _report_diff(contents, new_contents, filename):
     print(diff, end='')
 
 
-FUTURE_IMPORTS = (
-    ('py22', ('nested_scopes',)),
-    ('py23', ('generators',)),
-    ('py26', ('with_statement',)),
+VERSION_IMPORTS = (
+    ('py22', ('nested_scopes',), ''),
+    ('py23', ('generators',), ''),
+    ('py26', ('with_statement',), ''),
     (
         'py3',
         ('division', 'absolute_import', 'print_function', 'unicode_literals'),
+        '',
     ),
-    ('py37', ('generator_stop',)),
-)  # type: Tuple[Tuple[str, Tuple[str, ...]], ...]
+    (
+        'py33',
```

Comment:
I'm kind of leaning towards merging this into py3-plus -- I don't really want to introduce an argument for a version which is long since dead I guess (yeah yeah, there's existing py22 and py23 which are nonsense as well) hmmm

Reply:
Wasn't sure what the desired outcome was here, should I merge it into py3-plus after all?

Reply:
yeah let's put it into py3-plus